certif/app/controllers/authenticated/sessions/finalize.js

```diff
@@ -37,6 +37,10 @@ export default class SessionsFinalizeController extends Controller {
     return this.uncheckedHasSeenEndTestScreenCount > 0;
   }
 
+  get shouldDisplayStepTwo() {
```

Comment:
Il y a dans tout les cas un step 2, c'est le step "form" qui apparait conditionnellement
```suggestion
  get shouldDisplayFormStep() {
```

Reply:
Done